### PR TITLE
qa/tasks/ceph: raise exceptions if scrubbing fails or cannot proceed

### PR DESCRIPTION
**Pull request by liewegas**

### CODE REVIEW

**Comment by tchaikov** [comment]
retest this please (jenkins failure)

**Comment by liewegas** [comment]
scrub_test.yaml shoudl skip scrub.

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-05-28_05:00:18-rados-wip-sage-testing---basic-smithi/1238310

and end of thrasher, need to wait for osds to be up, then flush pg stats, then wait for healthy.

**Comment by liewegas** [comment]
retest this please

File: qa/tasks/ceph_manager.py [hunk 1]
```diff
@@ -1186,6 +1186,9 @@ def flush_pg_stats(self, osds, no_wait=None, wait_for_mon=3*5):
                                 'osd.{osd}: {got} < {need}'.
                                 format(osd=osd, got=got, need=need))
 
+    def flush_all_pg_stats(self):
+        self.flush_pg_stats(range(0, len(self.get_osd_dump())))
```
Comment (tchaikov) [inline qa/tasks/ceph_manager.py]: nit, `range(0, n)` is equivalent to `range(n)`

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-06-15_16:07:30-rados:thrash-master---basic-smithi/

**Comment by liewegas** [comment]
http://pulpito.ceph.com/sage-2017-06-16_00:46:50-rados-wip-sage-testing-distro-basic-smithi/

**Pull request merged**